### PR TITLE
docs(ava-react):improve readme of ava-react.zh and ava-react.en 

### PR DESCRIPTION
**Pull request by chenying2929**
add introduction and examples.

### CODE REVIEW

**Review by pddpd — COMMENTED** [review]
Why should we split them into two PRs? It feels like two commits or even one commit is enough.

#710 
#711

**Comment by chenying2929** [comment]
> Why should we split them into two PRs? It feels like two commits or even one commit is enough.
> 
> #710 #711

I am the first to change the Chinese document, submit the pr and wait for the audit to pass, and then modify the English version based on the Chinese version that has been changed, so generally the English and Chinese documents are modified separately to mention the PR. Do I need to merge them into the same pr this time ? ～

**Comment by pddpd** [comment]
> > Why should we split them into two PRs? It feels like two commits or even one commit is enough.
> > #710 #711
> 
> I am the first to change the Chinese document, submit the pr and wait for the audit to pass, and then modify the English version based on the Chinese version that has been changed, so generally the English and Chinese documents are modified separately to mention the PR. Do I need to merge them into the same pr this time ? ～

I think we could merge them into the current PR, and squash two commits into one.

File: packages/ava-react/zh-CN/README.zh-CN.md [hunk 1]
```diff
@@ -13,10 +13,18 @@ AVA 的 React 组件库
...
-## 组件
-- NarrativeTextVis
+## 简介
 
-## 安装
+[@antv/ava-react](https://www.npmjs.com/package/@antv/ava-react) 是基于 AVA 能力整合而成的即插即用的 React 组件库， 它包含三大核心组件：
+
+* <span style="background-color:#A9A9A9; padding:2px 4px; border-radius:4px;color: black;">\<NarrativeTextVis \/></span>：展示数据洞察解读文本。在数据分析的全流程展示中，使用文本描述数据现象和给出洞察结论同样关键。
+* <span style="background-color:#A9A9A9; padding:2px 4px; border-radius:4px;color: black;">\<InsightCard \/></span>：以图文结合的方式展示数据洞察。既可以直接接收数据并自动进行洞察，也可以仅对洞察结果数据进行可视化和解读的展示。
+* <span style="background-color:#A9A9A9; padding:2px 4px; border-radius:4px;color: black;">\<AutoChart \/></span>>：根据数据自动推荐合适的图表并渲染。它可以为用户提供一行代码实现智能可视化的能力，目前该组件正在进一步升级中。
```
Comment (pddpd) [inline packages/ava-react/zh-CN/README.zh-CN.md:22]: Have you previewed these mds? It seems like there are some problems.
image

**Review by pddpd — CHANGES_REQUESTED** [review]

File: packages/ava-react/zh-CN/README.zh-CN.md (hunk 1 above)
Comment (chenying2929) [reply packages/ava-react/zh-CN/README.zh-CN.md]: 1. The left slider, which is not shown in the PREVIEW on github.
image

2. The extra > symbol, which has been removed

**Review by pddpd — APPROVED** [review]
LGTM

**Pull request merged**